localization/ekf_localizer/config/ekf_localizer.param.yaml

```diff
@@ -22,6 +22,11 @@
     proc_stddev_vx_c: 10.0
     proc_stddev_wz_c: 5.0
 
+    #Simple1DFilter parameters
+    z_filter_proc_dev: 1.0
+    roll_filter_proc_dev: 0.01
+    pitch_filter_proc_dev: 0.01
+
```

Comment:
Please add these parameters in Readme as well

Reply:
@kminoda . Could you check again